src/eth1/impl/ethers.ts

```diff
@@ -125,6 +128,22 @@ export class EthersEth1Notifier extends EventEmitter implements Eth1Notifier {
     await this.db.deleteGenesisDeposits(genesisDeposits);
   }
 
+
+  public async getContractDeposits(
+    fromBlock: string | number  = this.opts.depositContract.deployedAt,
+    toBlock?: string | number
```

Comment:
I would default this to `null` so you don't need to pass in null

Reply:
it is null by default. Notice the question mark